lib/index.ts

```diff
@@ -109,6 +109,30 @@ const sdk = (...args: Parameters<typeof createSdk>) => {
       /* istanbul ignore next */
       throw Error('could not validate token')
     },
+
+    async exchangeAccessKey(accessKey: string): Promise<AuthenticationInfo> {
+      let resp: SdkResponse<any>
```

Comment:
```suggestion
      let resp: ReturnType<typeof this.accessKey.exchange>
```

Reply:
I agree that it should be clear for the next person what is the type we are using 
but `SdkResponse<any>` is not the correct type so we need to update it to match the `core-js` type
and this leads to the second problem, what if the `core-js` type will change? 
we will probably forget to update it so I think it's better using the original type

we can extract it to a new type so when hovering it we will see exactly which attributes it includes

WDYT?